test/specs/practise/PracticeCodeNFT.spec.js

```diff
@@ -1,69 +1,716 @@
 import * as dotenv from "dotenv";
```

Comment:
As with the previous file, can i just check what this "practise" file is for? Should it be skipped?

Reply:
The practice file is for practice purposes, So I initially write the code in the practice file and then implement it for each network as smoke and regression test cases.